community/py-markdown/APKBUILD

```diff
@@ -2,8 +2,8 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-markdown
 _pkgname=Markdown
-pkgver=3.0.1
-pkgrel=1
+pkgver=3.1
+pkgrel=0
 pkgdesc="A Python implementation of Markdown"
 url="http://pythonhosted.org/Markdown/"
 arch="noarch"
```

Comment:
makedepends only need py-setuptools and py3-setuptools

Reply:
I think this needs separate issue cos a lot of packages affected